development/style-guide.md

```diff
@@ -145,60 +139,92 @@ function bank_Account() {
 }
 ```
 
-## Use UPPERCASE for Constants
+## Use `const` and `let`
+
+There is no longer a good reason to use `var`. Use `const` whenever you can,
+and `let` when you must. Hardcoded constants should be named in all UPPERCASE.
+
+*Right:*
+
+```js
+const DELAY = 10 * 1000; // ten seconds
+const output = input * 10;
+let temp = 50;
+let unknown;
+```
+
+*Wrong:*
+
+```js
+var DELAY = 10 * 1000;
+```
+
+## Use arrow functions
```

Comment:
Is it worth mentioning the three (there may be more) good reasons to use `function`? That is:
 - A need for the `arguments` variable
 - A need for `this`
 - In `webapp` and you want to have step debugging

Reply:
Done.

Reply:
Not always appropriate, but just in case you haven't seen this: you can use the spread operator to get all arguments passed into a lambda. `(...args) => returns args.join(',');`. I find this useful, so just spreading the joy.